packages/experimental/src/experimental-list/task-item.tsx

```diff
@@ -114,9 +114,11 @@ export const TaskItem: React.FC< TaskItemProps > = ( {
 	actionLabel,
 	...listItemProps
 } ) => {
+	const [ isTaskExpanded, setTaskExpanded ] = useState( expanded );
+
 	const className = classnames( 'woocommerce-task-list__item', {
 		complete: completed,
-		expanded,
+		isTaskExpanded,
```

Comment:
We mostly use dash case for class names, so maybe change this to: `expanded: isTaskExpanded`

Reply:
Nice catch. This was fixed in the commit e4fb83c